doc/source/deploy/server/ami.rst

```diff
@@ -397,4 +397,141 @@ for the corresponding image:
+   increased. From the ``dmesg`` output, you should see something like below::
+
+     [  535.994494] xvdb: detected capacity change from 17179869184 to 34359738368
+
+3. You can check information about logical volumes and volume groups with the
+   ``lvs`` and ``vgs`` commands::
+
+    [root@ip-10-1-2-70 ~]# vgs
+     VG     #PV #LV #SN Attr   VSize VFree
+     DataVG   1 2 0 wz--n- <16.00g    0
+
+    [root@ip-10-1-2-70 ~]# lvs
+     LV       VG Attr       LSize Pool Origin Data%  Meta% Move Log Cpy%Sync Convert
+     DataLV   DataVG Vwi-aotz-- <12.80g ThinPool        1.74
+     ThinPool DataVG twi-aotz--  15.96g 1.39  1.29|
+
+4. Now we can use the ``lsblk`` command to see that our disk size has been read
+   by the kernel properly but LVM2 does not know about it yet. We can use
+   ``pvresize`` to make sure the PV device reflects the new size. Once
+   ``pvresize`` is executed, we can see that the VG has the new free space
+   available.
+
+   .. code-block:: bash
+
+      [root@ip-10-1-2-70 ~]# lsblk | grep xvdb
+       xvdb                      202:16 0 32G 0 disk
+
+      [root@ip-10-1-2-70 ~]# pvscan
+       PV /dev/xvdb   VG DataVG    lvm2 [<16.00 GiB / 0    free]
+       Total: 1 [<16.00 GiB] / in use: 1 [<16.00 GiB] / in no VG: 0 [0   ]
+
+      [root@ip-10-1-2-70 ~]# pvresize /dev/xvdb
+       Physical volume "/dev/xvdb" changed
+       1 physical volume(s) resized / 0 physical volume(s) not resized|
+
+      [root@ip-10-1-2-70 ~]# pvs
+       PV         VG Fmt Attr PSize   PFree
+       /dev/xvdb  DataVG lvm2 a--  <32.00g 16.00g
+
+5. We can then extend our logical volumes. Since the PMM image uses thin
+   provisioning, we need to extend both the pool and the volume::
+
+      [root@ip-10-1-2-70 ~]# lvs
+       LV       VG Attr       LSize Pool Origin Data%  Meta% Move Log Cpy%Sync Convert
+       DataLV   DataVG Vwi-aotz-- <12.80g ThinPool        1.77
+       ThinPool DataVG twi-aotz--  15.96g 1.42  1.32
+
+      [root@ip-10-1-2-70 ~]# lvextend /dev/mapper/DataVG-ThinPool -l 100%VG
+       Size of logical volume DataVG/ThinPool_tdata changed from 16.00 GiB (4096 extents) to 31.96 GiB (8183 extents).
+       Logical volume DataVG/ThinPool_tdata successfully resized.
+
+      [root@ip-10-1-2-70 ~]# lvs
+       LV       VG Attr       LSize Pool Origin Data%  Meta% Move Log Cpy%Sync Convert
+       DataLV   DataVG Vwi-aotz-- <12.80g ThinPool        1.77
+       ThinPool DataVG twi-aotz--  31.96g 0.71  1.71
+
+6. On the previous step, we can see our thin pool is now reflecting the whole
+   size of the disk, next is we need to extend the thin volume as well close to
+   available pool capacity.
+
+   .. code-block:: bash
+
+      [root@ip-10-1-2-70 ~]# lvs
+       LV       VG Attr       LSize Pool Origin Data%  Meta% Move Log Cpy%Sync Convert
+       DataLV   DataVG Vwi-aotz-- <12.80g ThinPool        1.77
+       ThinPool DataVG twi-aotz--  31.96g 0.71  1.71
+
+      [root@ip-10-1-2-70 ~]# lvextend /dev/mapper/DataVG-DataLV -L +19G
```

Comment:
@fiowro How did you know to use 19G here? we should explain how this value is arrived at

Reply:
updated text with clarification of this moment.